plugins/Eq/EqControlsDialog.cpp

```diff
@@ -45,113 +45,287 @@ EqControlsDialog::EqControlsDialog( EqControls *controls ) :
-		{
-			m_freqKnob->move( cw * i + ko, 222 );
-		} else
-		{
-			m_freqKnob->move(cw * i + ko, 235 );
-		}
+		MLayout->addWidget( m_freqKnob, 3, i+1 );
+		MLayout->setAlignment( m_freqKnob, Qt::AlignHCenter );
 		m_freqKnob->setVolumeKnob( false );
 		m_freqKnob->setModel( m_parameterWidget->getBandModels( i )->freq );
 		m_freqKnob->setHintText( tr( "Frequency:" ), "Hz" );
 
-		m_activeBox = new LedCheckBox( m_parameterWidget->getBandModels( i )->name , this , "" , LedCheckBox::Green );
-		m_activeBox->move( cw * i + fo + 3, 260 );
+		// adds the Number Active buttons
+		m_activeBox = new PixmapButton( this, NULL );
+		m_activeBox->setCheckable(true);
+		m_activeBox->setModel( m_parameterWidget->getBandModels( i )->active );
+
+		switch (i)
+		{
+		case 0:
+			m_activeBox->setActiveGraphic( PLUGIN_NAME::getIconPixmap(
+											   "bandLabel1on" ) );
+			m_activeBox->setInactiveGraphic( PLUGIN_NAME::getIconPixmap(
+												 "bandLabel1" ) );
+			break;
+		case 1:
+			m_activeBox->setActiveGraphic( PLUGIN_NAME::getIconPixmap(
+											   "bandLabel2on" ) );
+			m_activeBox->setInactiveGraphic( PLUGIN_NAME::getIconPixmap(
+												 "bandLabel2" ) );
+			break;
+		case 2:
+			m_activeBox->setActiveGraphic( PLUGIN_NAME::getIconPixmap(
+											   "bandLabel3on" ) );
+			m_activeBox->setInactiveGraphic( PLUGIN_NAME::getIconPixmap(
+												 "bandLabel3" ) );
+			break;
+		case 3:
+			m_activeBox->setActiveGraphic( PLUGIN_NAME::getIconPixmap(
+											   "bandLabel4on" ) );
+			m_activeBox->setInactiveGraphic( PLUGIN_NAME::getIconPixmap(
+												 "bandLabel4" ) );
+			break;
+		case 4:
+			m_activeBox->setActiveGraphic( PLUGIN_NAME::getIconPixmap(
+											   "bandLabel5on" ) );
+			m_activeBox->setInactiveGraphic( PLUGIN_NAME::getIconPixmap(
+												 "bandLabel5" ) );
+			break;
+		case 5:
+			m_activeBox->setActiveGraphic( PLUGIN_NAME::getIconPixmap(
+											   "bandLabel6on" ) );
+			m_activeBox->setInactiveGraphic( PLUGIN_NAME::getIconPixmap(
+												 "bandLabel6" ) );
+			break;
+		case 6:
+			m_activeBox->setActiveGraphic( PLUGIN_NAME::getIconPixmap(
+											   "bandLabel7on" ) );
+			m_activeBox->setInactiveGraphic( PLUGIN_NAME::getIconPixmap(
+												 "bandLabel7" ) );
+			break;
+		case 7:
+			m_activeBox->setActiveGraphic( PLUGIN_NAME::getIconPixmap(
+											   "bandLabel8on" ) );
+			m_activeBox->setInactiveGraphic( PLUGIN_NAME::getIconPixmap(
+												 "bandLabel8" ) );
+			break;
+		}
+
+
+		MLayout->addWidget( m_activeBox, 1, i+1 );
+		MLayout->setAlignment( m_activeBox, Qt::AlignHCenter );
+		m_activeBox->setModel( m_parameterWidget->getBandModels( i )->active );
+
+		// adds the symbols active buttons
+		m_activeBox = new PixmapButton( this, NULL );
+		m_activeBox->setCheckable(true);
+		m_activeBox->setModel( m_parameterWidget->getBandModels( i )->active );
+		switch (i)
+		{
+		case 0:
+			m_activeBox->setActiveGraphic( PLUGIN_NAME::getIconPixmap(
```

Comment:
Minor formatting, but these 16 `>setActiveGraphic(...)` lines can easily be consolidated into single-line and I'm not sure a line break is warranted before the `break;`